src/data/set/intervals/interval.lean

```diff
@@ -0,0 +1,137 @@
+/-
+Copyright (c) 2020 Zhouhang Zhou. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Zhouhang Zhou
+-/
+
+import data.set.intervals
+
+/-!
+# Intervals
```

Comment:
I think some clarification is needed in the title to distinguish it from the other file.
Unordered intervals? Intervals without endpoints ordering?